specs/capitalize_decorator_spec.rb

```diff
@@ -0,0 +1,13 @@
+require_relative '../decorator'
+require_relative '../capitalize_decorator'
```

Comment:
- [ ] Kindly add tests for all classes that you created while implementing the Decorator design pattern as this is a requirement.
Please add tests for the `nameable..rb` class

![image](https://github.com/Munyabelden/Oop_Library/assets/81932478/3af5d913-422d-4c52-8d96-1d0568a0582d)